helm-charts/helm3/strimzi-kafka-operator/crds/040-Crd-kafka.yaml

```diff
@@ -1,15 +1,10 @@
-{{- if .Values.createGlobalResources -}}
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: kafkas.kafka.strimzi.io
   labels:
-    app: '{{ template "strimzi.name" . }}'
```

Comment:
I think the app label is fairly useful to find all resources belonging to the app ... I wonder if we should find a way to add it back. But not sure what the options are if the templates are not supported here.

Reply:
Previously, the default value here is "strimzi" as set in `_helpers.tpl`. I was able to find 2 examples where charts that define the CRDs in the `crds` directory apply an app label. Should I add the app label back and set it to "strimzi"?

https://github.com/helm/charts/blob/master/stable/etcd-operator/crds/etcd-cluster-crd.yaml#L8
https://github.com/helm/charts/blob/master/stable/jaeger-operator/crds/crd.yaml#L9

I guess this could be a breaking change if a user is looking at these labels for something. I can't think of a use case though where someone would do that for CRDs.

Reply:
TBH, I find it useful. But I'm also fine to follow the Helm conventions which I don't know too much.

Reply:
I also guess it makes no sense to have here hardcoded `app: strimzi` while in the other resources to have configurable `app: something-else`.

Reply:
@scholzj @sgkiokas After running `make crd_install` the app label was added back. I had originally edited the CRDs directly not understand that they were generated by the makefile. Do we have a consensus on whether to keep the app label or should I update the makefile to remove the app label?

Reply:
I think you said on the beginning that _templates are not support in the `crds` directory for Helm 3_ ... so to be honest it is not clear to me if keep the `app` label actually works or not. If it works I would keep it I guess.

Reply:
This wouldn't be a template. The app label would be hard coded to the value "strimzi".

Reply:
Ah, ok. I think that makes really sense only if we hardcode the `app` label to `strimzi` in the other resources as well. Otherwise I see little value in it. Is it common in other Helm Charts to have the `app` label configurable?

Reply:
I'm in agreement here, the app label is not really necessary, since we can easily find them due to having a deterministic resource group.

Whichever is easiest from a generation perspective, either keeping the app label or not adding the label is what I suggest we do here.

Reply:
> Whichever is easiest from a generation perspective, either keeping the app label or not adding the label is what I suggest we do here.

@samuel-hawker Shouldn't be difficult to write a `yq` command to remove the app label but easiest is to leave it.

Reply:
Yes leave it for now and we can remove it if there is sensible justification for it.